recipes/mehari/build.sh

```diff
@@ -7,6 +7,9 @@ export CFLAGS="${CFLAGS} -O3 -Wno-implicit-function-declaration -Wno-int-convers
 # Make sure bindgen passes on our compiler flags.
 export BINDGEN_EXTRA_CLANG_ARGS="${CPPFLAGS} ${CFLAGS} ${LDFLAGS}"
 
+sed -i.bak 's|"0.23"|"0.24.0"|' Cargo.toml
```

Comment:
@mencian I'd appreciate if you could request this change (that I guess currently targets rocksdb, but may also hit others in the future) upstream instead. And I guess it'd be better if it were 0.23.0 for now